api/app.proto

```diff
@@ -69,609 +88,805 @@ message UploadAppAttachmentRequest {
 	google.protobuf.StringValue screenshots = 9;
+	// maintainers of app of specific version
 	google.protobuf.StringValue maintainers = 10;
+	// keywords of app of specific version
 	google.protobuf.StringValue keywords = 11;
+	// sources of app of specific version
 	google.protobuf.StringValue sources = 12;
+	// readme of app of specific version
 	google.protobuf.StringValue readme = 13;
+	// package name of app of specific version
 	google.protobuf.StringValue package_name = 14;
+	// status of app of specific version, eg.[active|deleted]
 	google.protobuf.StringValue status = 15;
+	// review id of app of specific version
 	google.protobuf.StringValue review_id = 16;
+	// the time when app version create
 	google.protobuf.Timestamp create_time = 17;
+	// record status changed time
 	google.protobuf.Timestamp status_time = 18;
+	// the time when app version update
 	google.protobuf.Timestamp update_time = 19;
+	// sequence of app of specific version
 	google.protobuf.UInt32Value sequence = 20;
+	// message path of app of specific version
 	google.protobuf.StringValue message = 21;
+	// type of app of specific version
 	google.protobuf.StringValue type = 22;
+	// owner path of app of specific version, concat string group_path:user_id
 	google.protobuf.StringValue owner_path = 23;
 }
 
 message AppVersionAudit {
+	// id of version to audit
 	google.protobuf.StringValue version_id = 1;
+	// id of specific version app
 	google.protobuf.StringValue app_id = 2;
+	// audit status
 	google.protobuf.StringValue status = 3;
+	// version name
 	google.protobuf.StringValue version_name = 4;
+	// name of specific version app
 	google.protobuf.StringValue app_name = 5;
+	// user of auditer
 	google.protobuf.StringValue operator = 6;
+	// user role of auditer
 	google.protobuf.StringValue operator_type = 7;
+	// audit message
 	google.protobuf.StringValue message = 8;
+	// review id
 	google.protobuf.StringValue review_id = 9;
+	// record status changed time
 	google.protobuf.Timestamp status_time = 10;
+	// version type
 	google.protobuf.StringValue version_type = 11;
 }
 
 message AppVersionReviewPhase {
+	// review status of app version
 	google.protobuf.StringValue status = 1;
+	// reviewer of the special app version
 	google.protobuf.StringValue operator = 2;
+	// review role of the special version app
 	google.protobuf.StringValue operator_type = 3;
+	// review message
 	google.protobuf.StringValue message = 4;
+	// record status changed time
 	google.protobuf.Timestamp status_time = 5;
+	// app version review time
 	google.protobuf.Timestamp review_time = 6;
 }
 
 message AppVersionReview {
+	// review id
 	google.protobuf.StringValue review_id = 1;
+	// id of app version
 	google.protobuf.StringValue version_id = 2;
+	// app id
 	google.protobuf.StringValue app_id = 3;
+	// version name of specific app version
 	google.protobuf.StringValue version_name = 4;
+	// app name
 	google.protobuf.StringValue app_name = 5;
+	// review status
```

Comment:
[isv-in-review|isv-passed|isv-rejected|isv-draft|business-in-review|business-passed|business-rejected|develop-draft|develop-in-review|develop-passed|develop-rejected|develop-draft]